### ISSUE TITLE
SCSS-Parser fails to handle triple slash comments with a quote

### ISSUE THREAD

**Issue by MattDiMu** [stylelint/stylelint#1097]
How to reproduce:
.1. Create the following single line SCSS-File using CRLF line endings
```scss
/// this is a 75-char-comment causing problems due to it's quote character
```
.2. lint it using the standard stylelint config ("string-no-newline"-rule should be checked)
```
stylelint _test.scss --syntax=scss
 1:78  ×  Unexpected newline in string   string-no-newline
```

.3. Convert it to LF line endings and run again
```
stylelint _test.scss --syntax=scss
1:77  ×  Unexpected newline in string   string-no-newline
```

I spot 2 different problems:
1. the SCSS-Parser fails to recognize the line as SCSS-Comment, if there are 3 slashes (e.g. necessary for https://github.com/SassDoc/sassdoc commenting). There should never be an error be reported at all, as the quote is in a comment!
2. If there is an error, it should be the same char-number (77 != 78) and NOT greater than the total amount of characters in this line (75!).

This (wrong char-number) causes atom-stylelint to break, just as recently fixed #1081 did.

edit: Just to be clear: The problem exists in both cases (CRLF and LF files), both char-numbers are wrong.

**Comment by davidtheclark** [comment]
Unfortunately I don't have time to track this down this morning. Any additional help appreciated!
